redun/file.py

```diff
@@ -677,7 +703,11 @@ class GSFileSystem(FsspecFileSystem):
     name = "gs"
 
     def shell_copy(
-        self, src_path: Optional[str], dest_path: Optional[str], recursive: bool = False
+        self,
+        src_path: Optional[str],
+        dest_path: Optional[str],
+        recursive: bool = False,
+        as_mount: bool = False,
```

Comment:
I think I am generally onboard with approach. I am also looking at whether we want this mounting approach be staging concept instead of a copying concept, but so far I think this is fine.